cmd/boost/deal_cmd.go

```diff
@@ -39,9 +39,11 @@ var dealCmd = &cli.Command{
 			Value: "~/.boost-client",
 		},
 		&cli.StringFlag{
-			Name:     "http-url",
-			Usage:    "http url to CAR file",
-			Required: true,
+			Name:  "http-url",
+			Usage: "http url to CAR file",
+		},
+		&cli.BoolFlag{
+			Name: "offline",
```

Comment:
The usage doc says "make an online deal with Boost". Don't you think best to not overload this command and best top have a separate command for offline deals ?